libcodechecker/libclient/authentication_helper.py

```diff
@@ -20,10 +20,12 @@
 
 
 class ThriftAuthHelper():
-    def __init__(self, host, port, uri, session_token=None):
+    def __init__(self, protocol, host, port, uri,
+                 session_token=None):
         self.__host = host
         self.__port = port
-        self.transport = THttpClient.THttpClient(self.__host, self.__port, uri)
+        url = protocol+"://"+host+":"+str(port)+uri
+        self.transport = THttpClient.THttpClient(url)
         self.protocol = TJSONProtocol.TJSONProtocol(self.transport)
         self.client = codeCheckerAuthentication.Client(self.protocol)
```

Comment:
The `protocol` and `self.protocol` are two distinct things. Maybe we should name them separately.